internal/collector/telemetry/telemetry.go

```diff
@@ -91,8 +91,8 @@ func Flags(flags *flag.FlagSet) {
 // the current build type.
 func GetMetricsAddrDefault() string {
 	if version.IsDevBuild() {
-		// Listen on localhost by default for dev builds to avoid security prompts.
-		return "localhost:8888"
+		// Listen on 0.0.0.0 by default for dev builds to avoid security prompts.
+		return "0.0.0.0:8888"
```

Comment:
This should stay localhost as it was intentionally added to default to localhost for dev builds.